storage/innobase/os/os0file.cc

```diff
@@ -1990,6 +1990,10 @@ static char *os_file_get_parent_dir(const char *path) {
     return (NULL);
   }
 
+  if (last_slash - path < 0) {
```

Comment:
what does this code mean/ does? both are char pointers

Reply:
Please add comment (like it was done above) :)

Reply:
done